src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -618,10 +618,17 @@
   <data name="RenameFailedToast.Subtitle" xml:space="preserve">
     <value>Another window with that name already exists</value>
   </data>
+  <data name="RenameSucceededToast.Title" xml:space="preserve">
+    <value>Successfully renamed window</value>
+  </data>
+  <data name="RenameSucceededText" xml:space="preserve">
+    <value>Successfully renamed window to "{}"</value>
+    <comment>{} will be replaced by the new name assigned by the user</comment>
+  </data>

```

Comment:
This seems a bit redundant. @cinnamon-msft mind taking a look?

Reply:
Yeah I was just thinking we could pop up the same toast as the `identifyWindow` command. It does the same thing without having to localize more stuff. :)

Reply:
this is a _great_ idea